core/internals.h

```diff
@@ -107,14 +107,41 @@
 ((S) == STATE_BS_FAILING ? "STATE_BS_FAILING" :                 \
 ((S) == STATE_BS_FAILED ? "STATE_BS_FAILED" :                   \
 "Unknown"))))))))))))))))
-#define STR_MEDIA_TYPE(M)                                        \
-((M) == LWM2M_CONTENT_TEXT ? "LWM2M_CONTENT_TEXT" :              \
-((M) == LWM2M_CONTENT_LINK ? "LWM2M_CONTENT_LINK" :              \
-((M) == LWM2M_CONTENT_OPAQUE ? "LWM2M_CONTENT_OPAQUE" :          \
-((M) == LWM2M_CONTENT_TLV ? "LWM2M_CONTENT_TLV" :                \
-((M) == LWM2M_CONTENT_JSON ? "LWM2M_CONTENT_JSON" :              \
-((M) == LWM2M_CONTENT_SENML_JSON ? "LWM2M_CONTENT_SENML_JSON" :  \
-"Unknown"))))))
+#define STR_MEDIA_TYPE(M)                                                                                              \
+    ((M) == LWM2M_CONTENT_TEXT                                                                                         \
+         ? "LWM2M_CONTENT_TEXT"                                                                                        \
+         : ((M) == LWM2M_CONTENT_LINK                                                                                  \
+                ? "LWM2M_CONTENT_LINK"                                                                                 \
+                : ((M) == LWM2M_CONTENT_OPAQUE                                                                         \
+                       ? "LWM2M_CONTENT_OPAQUE"                                                                        \
+                       : ((M) == LWM2M_CONTENT_TLV                                                                     \
+                              ? "LWM2M_CONTENT_TLV"                                                                    \
+                              : ((M) == LWM2M_CONTENT_JSON                                                             \
+                                     ? "LWM2M_CONTENT_JSON"                                                            \
+                                     : ((M) == LWM2M_CONTENT_SENML_JSON                                                \
+                                            ? "LWM2M_CONTENT_SENML_JSON"                                               \
+                                            : ((M) == LWM2M_CONTENT_CBOR                                               \
+                                                   ? "LWM2M_CONTENT_CBOR"                                              \
+                                                   : ((M) == LWM2M_CONTENT_SENML_CBOR ? "LWM2M_CONTENT_SENML_CBOR"     \
+                                                                                      : "Unknown"))))))))
+
+/* clang-format off */
```

Comment:
Move `/* clang-format off */` up, so that this uglification does not happen.

Reply:
Done